### PR TITLE
Add forms

### PR DESCRIPTION
**Pull request by imubarak234**
Create forms to perform the following functions:

Creates a Post on behalf of the current_user.
Create a comment on behalf of the current_user.
Allow Users to add likes to Posts.

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -1,9 +1,14 @@
 Rails.application.routes.draw do
+
   resources :users, only: [:index, :show] do
     resources :posts, only: [:index, :show, :new, :create] do
       resources :comments, only: [:index, :new, :create]
     end
   end
+
```
Comment (andersonlebon) [inline config/routes.rb]: - [ ] KIndly define a default route(root) where your application should start running. Hint :point_down: 

```rb
root "users#index"
```

File: app/controllers/posts_controller.rb [hunk 2]
```diff
@@ -42,7 +38,7 @@ def create
       format.html do
         if post_create.save
           flash[:success] = 'Post created successfully'
-          redirect_to user_posts_url
+          redirect_to user_post_url
```
Comment (andersonlebon) [inline app/controllers/posts_controller.rb]: - [ ]  Kindly note that this is a wrong syntax in rails it is raising this error :point_down: 

image

User, this to redirect correctly to the new post created.

```rb
          redirect_to user_post_path(current_user_id, post_create.id)

```

**Review by andersonlebon — COMMENTED** [review]
## Hi @imubarak234 

Good job so far! I know you have worked hard on making this milestone But  There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there
## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
\- Good PR description :heavy_check_mark: 
## Required Changes ♻️

- [ ] Kindly check comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by imubarak234** [comment]
For the redirect, my error message suggested an 's' was missing 'redirect_to user_post_url' needed to be 'redirect_to user_posts_url'. This change seems to fix it 👍

**Review by dasileker — APPROVED** [review]
### **Hi @imubarak234,** 

### **Congratulations! 🎉**

Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀 : 
 

### **Optional**

- The readme file  `git clone git@https://github.com/imubarak234/rails-blogger.git
cd rails-blogger
To Start the programs
open terminal
run rails s` it will not make the app run correctly so please add how to setup database (create and migrate) to make the reader understand how to proceed.


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**